robottelo/vm.py

```diff
@@ -212,27 +212,6 @@ def install_katello_agent(self):
         if result.return_code != 0:
             raise VirtualMachineError('Failed to install katello-agent')
 
-    def install_katello_cert(self):
```

Comment:
Makes sence for me :+1: 
I don't like how it looks right now, as for vm's we don't have a valid hostname and are forced to pass ip address to hostname argument which is ugly. Leaving a helper will resolve that.